src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -46,6 +47,43 @@ const overview = async function overview(req: Request, res: Response): Promise<v
   })
 }
 
+const listCSV = async function listCSV(req: Request, res: Response): Promise<void> {
+  const servicesRequest = AdminUsers.services()
+  const accountsRequest = Connector.accounts().then((response: any) => response.accounts)
+
+  const [serviceResponse, accountsResponse] = await Promise.all([servicesRequest, accountsRequest])
+
+  const serviceGatewayAccountIndex = serviceResponse
```

Comment:
Good idea, I'll do this. Where do you think this best belongs in the package hierarchy?

Reply:
How about `src/lib/gatewayAccounts` ? Slightly confusing as they also come from `Connector` but it could be named clearly enough to show that it was getting data from multiple services and putting it together

Reply:
🍬 I can think of about 4 places we can clean up the code with that in!